src/EA.Weee.Web/Areas/Admin/Mappings/ToViewModel/AatfDataToAatfDetailsViewModelMap.cs

```diff
@@ -44,7 +44,10 @@ public AatfDetailsViewModel Map(AatfDataToAatfDetailsViewModelMapTransfer source
                 SiteAddressLong = addressUtilities.FormattedAddress(source.AatfData.SiteAddress, false),
                 ContactAddressLong = addressUtilities.FormattedAddress(source.AatfData.Contact.AddressData, false),
                 PanArea = source.AatfData.PanAreaData,
-                LocalArea = source.AatfData.LocalAreaData
+                LocalArea = source.AatfData.LocalAreaData,
+                ComplianceYearList = source.ComplianceYearList,
+                AatfId = source.AatfData.AatfId,
+                SelectedComplianceYear = source.AatfData.ComplianceYear
```

Comment:
Update test to check selected compliance year is set

Reply:
Done